index.js

```diff
@@ -14,7 +14,9 @@ module.exports = function (ms) {
 		});
 	}
 
-	thunk.then = promise.then.bind(promise);
+	['promise', 'catch'].forEach(function (method) {
+		thunk[method] = promise[method].bind(promise);
+	});
```

Comment:
Ok, I had figured that this way it was easier to add more later, but I'll change that.